Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -520,6 +520,8 @@ def rasterize_email_command():
     w, h = check_width_and_height(w, h)  # Check that the width and height meet the safeguard limit
 
     file_name = f'{file_name}.{"pdf" if r_type == RasterizeType.PDF else "jpeg"}'  # type: ignore
+    # force setting the charset in order to ovveride any posible wrong charsets
+    html_body = html_body.replace('<meta ', '<meta charset="UTF-8" ')
```

Comment:
What if the `meta` tag already contains `charset`? 
use regex to avoid conflicts